src/utilities/knativeEventing.ts

```diff
@@ -0,0 +1,8 @@
+import { emitterFor, type EmitterFunction, httpTransport } from 'cloudevents';
+import envVar from 'env-var';
+
+export function makeEmitterFromEnv(): EmitterFunction {
```

Comment:
This function isn't used in this PR, but will be used when we implement `MemberPublicKeyImport` in `POST /awala`. (I tested it by integrating it in the healthcheck, but I'm obviously not committing that change)